config/README.md

```diff
@@ -1,37 +1,70 @@
 # OPERATOR-CONTROLLER CONFIG
 
-## config/base
+## config/overlays/basic-olm
 
-This provides an insecure (i.e. no TLS) basic configuration of operator-controller.
-
-This configuration specifies a namespace of `olmv1-system`.
+This includes basic support for an insecure OLMv1 deployment. This configuration uses:
+* config/base/operator-controller
+* config/base/catalogd
+* config/base/common
 
 ## config/overlays/cert-manager
 
 This includes support for a secure (i.e. with TLS) configuration of operator-controller. This configuration uses:
-* config/base
-* config/components/tls
-* config/components/ca
+* config/base/operator-controller
+* config/base/catalogd
+* config/base/common
+* config/components/tls/operator-controller
+* config/components/tls/catalogd
+* config/components/tls/ca
 
 This configuration requires cert-manager.
 
 ## config/overlays/e2e
 
 This provides additional configuration support for end-to-end testing, including code coverage. This configuration uses:
-* config/base
-* config/components/tls
-* config/components/ca
+* config/base/operator-controller
+* config/base/catalogd
+* config/base/common
+* config/components/tls/operator-controller
+* config/components/tls/catalogd
 * config/components/coverage
+* config/components/tls/ca
```

Comment:
Nit: swap lines 30 and 31 so that all the `tls` components are grouped together.

Reply:
Can't. If you look in other places, there's a comment stating that it messes up the namespace of the CA components. I should put that comment here.

Reply:
But, here, since it's a readme, I suppose I could.